test/scenarios/functions/array_functions.jt

```diff
@@ -0,0 +1,4 @@
+{
+    map: .map(lambda ?0 * 2),
```

Comment:
why `0` suffix is needed in `?0` to denote the element in the list ? Is it any standard convention ?